### PR TITLE
add stealthwatch

### PR DESCRIPTION
**Pull request by CS-Amol**

### CODE REVIEW

File: stealthwatch/info.json [hunk 1]
```diff
@@ -0,0 +1,231 @@
+{
+  "name": "stealthwatch",
+  "label": "StealthWatch",
+  "description": "StealthWatch Connector",
```
Comment (satishmishra1901) [inline stealthwatch/info.json]: Meaningful description

File: stealthwatch/info.json [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        "visible": true,
+        "type": "text",
+        "name": "username"
+      },
+      {
+        "title": "Password",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
```
Comment (satishmishra1901) [inline stealthwatch/info.json]: type ```password``` . Its supported in 4.9 patch2

File: stealthwatch/info.json [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "end"
+        }
+      ],
+      "enabled": true,
+      "output_schema": [{"timePeriod" : "",
+        "applicationTrafficPerApplication" : [{
```
Comment (satishmishra1901) [inline stealthwatch/info.json]: lets have it formatted

File: stealthwatch/info.json [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+        "trafficWithinBps" : "",
+        "applicationName" : "",
+        "trafficInboundBps" : ""
+      }]}]
+
+    },
+    {
+      "operation": "get_domain_details",
+      "title": "Get All Domain Details",
+      "description": "Get All Domain Details for Get Domain ID",
```
Comment (satishmishra1901) [inline stealthwatch/info.json]: description ?

File: stealthwatch/operation.py [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+def get_domain_details(config, params, **kwargs):
+    try:
+        server_url , verify_ssl , cookie = get_config_params(config)
+        headers = {'Cookie': cookie }
+        url = server_url + api_all_domain
+        api_response = requests.get(url , headers=headers , verify=verify_ssl)
+        if api_response.ok:
+            return json.loads(api_response.content.decode('utf-8'))
+        else:
+            raise ConnectorError('get_domain_details() Fail %s', api_response.content.decode('utf-8'))
```
Comment (satishmishra1901) [inline stealthwatch/operation.py]: Message: message string , Status: status code

Comment (CS-Amol) [reply stealthwatch/operation.py]: same format is in api_response.content
like this : '{\"statusCode\":404,\"errorCode\":\"NOT_FOUND\"}'

**Pull request merged**